inc/common/purge.php

```diff
@@ -651,8 +651,18 @@ function rocket_clean_cache_theme_update( $wp_upgrader, $hook_extra ) {
  * @param array $post_data Array of unslashed post data.
  */
 function rocket_clean_post_cache_on_slug_change( $post_id, $post_data ) {
-	if ( get_post_field( 'post_name', $post_id ) !== $post_data['post_name'] ) {
-        rocket_clean_files( get_the_permalink( $post_id ) );
-    }
+	// Bail out if the post status is draft, pending or auto-draft.
+	if ( in_array( get_post_field( 'post_status', $post_id ), [ 'draft', 'pending', 'auto-draft' ], true ) ) {
+		return;
+	}
+	// Bail out if the slug hasn't changed.
+	if ( get_post_field( 'post_name', $post_id ) === $post_data['post_name'] ) {
+		return;
+	}
+	// Bail out if the old slug has changed, but is empty.
+	if ( empty( get_post_field( 'post_name', $post_id ) ) ) {
+		return;
+	}
+	rocket_clean_files( get_the_permalink( $post_id ) );
 }
 add_action( 'pre_post_update', 'rocket_clean_post_cache_on_slug_change', 10, 2 );
```

Comment:
@crystinutzaa 2 things:

1. Let's make our callback fire as late as possible by changing it from `10` to `PHP_INT_MAX`.  Why? By changing the priority number to the highest number possible, we are registering our callback in the last queue. That means ours will called after everything other 3rd party callback that has a priority number less than ours.
2. Plus, add an empty line at end of this file, ie for WPCS.